src/main/kotlin/mate/academy/CountTheMonkeys.kt

```diff
@@ -1,7 +1,5 @@
 package mate.academy
 
-// Implement the `countTheMonkeys()` function so that it returns an IntArray
-// with elements from `1` to `count`.
 fun countTheMonkeys(count: Int): IntArray {
-
+    return IntArray(count) { it + 1 }
```

Comment:
Cool solution) i don't expect it pass tests :) but checked its logic and now also understand it.